src/components/user-menu/user-menu.tsx

```diff
@@ -0,0 +1,60 @@
+import { NavLink } from '@remix-run/react';
+import classNames from 'classnames';
+import { useUserInfo } from '~/lib/hooks';
+import { Avatar } from '../avatar/avatar';
+import {
+    DropdownMenu,
+    DropdownMenuItem,
+    DropdownMenuSeparator,
+} from '../dropdown-menu/dropdown-menu';
+import { ChevronDownIcon } from '../icons';
+
+import styles from './user-menu.module.scss';
+
+export const UserMenu = () => {
+    const { isLoggedIn, user } = useUserInfo();
+
+    if (!isLoggedIn) {
+        return (
+            <NavLink className={classNames(styles.link, styles.root)} to={'/login'}>
+                <Avatar imageSrc={user?.profile?.photo?.url} />
```

Comment:
isn't this suppose to be anonymous user avatar?

Reply:
nice catch, missed this while refactoring...
lets keep `Avatar` component here, it handles `undefined` value correctly